### DISCUSSION TITLE
no mutation exists

### DISCUSSION THREAD

**Discussion by linklin830912** [hasura/graphql-engine#10609] [Question] [answered]
Hi all,
I'm new to hasura, I am setting up hasura with MongoDB according to the instruction here:https://www.mongodb.com/developer/products/atlas/graphql-apis-hasura/
While query works, I'm not sure why there's no mutation option in the interface: 
![image](https://github.com/user-attachments/assets/e5f7b96e-d592-4761-b2ae-77102fe2471a)
also when I wrote a mutation manually, it shows:
<img width="554" alt="image" src="https://github.com/user-attachments/assets/db972c55-9276-4dc7-9039-e90335a734ba">
is there any possible solutions?
Thanks

**Comment by praveenweb** [comment, marked as answer]
Hi @linklin830912 - The [MongoDB connector](https://hasura.io/docs/2.0/databases/mongodb/index/) for Hasura v2 only supports queries. 

We recommend upgrading to Hasura DDN and use the latest [Mongo Connector](https://github.com/hasura/ndc-mongodb/) to try out mutations. The tutorial you see on MongoDB will be updated soon with the change to Hasura DDN.
